### ISSUE TITLE
Performance of linear model using embedded features

### ISSUE THREAD

**Issue by macaodha** [Microsoft/snca.pytorch#2]
Hi there,

I really like the paper and was wondering if you tried training a linear SVM on your 128 dim feature space - and if so how will did it perform compared to the standard softmax approach. 

Thanks
